test/null-compatibility.spec.ts

```diff
@@ -0,0 +1,46 @@
+import {HyperFormula} from '../src'
+import {adr} from './testUtils'
+
+describe('null compatibility', () => {
+  it('reference - default behaviour', () => {
```

Comment:
Tests should describe what are the expectations.

If someone introduces a bug in test, this may still be accepted as the "default behaviour" because noone reading the code knows what it is